src/tlog-test-timespec.c

```diff
@@ -48,15 +48,16 @@ op_list_test(const struct op *op_list, size_t op_num,
         passed = (res.tv_sec == exp_res.tv_sec && res.tv_nsec == exp_res.tv_nsec);
         if (passed) {
             fprintf(stderr,
-                    "PASS " TLOG_TIMESPEC_FMT " %s " TLOG_TIMESPEC_FMT
+                    "[PASS] " TLOG_TIMESPEC_FMT " %s " TLOG_TIMESPEC_FMT
                     " == " TLOG_TIMESPEC_FMT "\n",
                     TLOG_TIMESPEC_ARG(&a), op_list[i].sym,
                     TLOG_TIMESPEC_ARG(&b),
                     TLOG_TIMESPEC_ARG(&exp_res));
         } else {
             fprintf(stderr,
-                    "FAIL " TLOG_TIMESPEC_FMT " %s " TLOG_TIMESPEC_FMT
+                    "[FAIL] %10s:%10d:%15s(): " TLOG_TIMESPEC_FMT " %s " TLOG_TIMESPEC_FMT
                     " = " TLOG_TIMESPEC_FMT " != " TLOG_TIMESPEC_FMT "\n",
+                    __FILE__,  line, __func__,
```

Comment:
I think we can do without `__func__` here, especially since it's always the same. Then, to be uniform, please pass the file name as an argument too. Also, please don't pad filename and line number separately. Instead join them with a colon in a separate buffer, and pad that buffer to the right in this `fprintf` here. That way we'll save some horizontal space.